### PR TITLE
Rules, Questions and Final Score pages

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
As part of this PR the following pages have been created :

- Rules Page : https://rules-and-questions.netlify.app/rules.html

rules-page

- Questions Page : https://rules-and-questions.netlify.app/questions.html

questions-page

- Score Card : https://rules-and-questions.netlify.app/score.html

score-card

### CODE REVIEW

File: styles/questions.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    text-align:left;
+    width:30rem;
+}
+
+.option-radio{
+    display:none;
+}
+
+.option-label{
+    background-color: var(--teritiary-clr);
```
Comment (venkateshwarans) [inline styles/questions.css]: tertiary spelling

File: styles/questions.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.option-label:hover{
+    background-color: var(--secondary-clr);
+    cursor:pointer;
+    transform: translate3D(0,-1px,0) scale(1.03);
+}
+
+.correct-answer{
+    background-color: var(--clr-green);
```
Comment (venkateshwarans) [inline styles/questions.css:28]: just nitpicking. `--secondary-clr` and `--clr-green`. missing consistency.

**Review by venkateshwarans — APPROVED** [review]
Overall great work. not to sound patronizing, but your coding is evolving PR by PR. :) 

One suggestion - leave consistence space between 
`property-name: value` . 💯

**Review by foolhardy21 — APPROVED** [review]
looks good to me

**Review by MayankKumar10 — APPROVED** [review]
LGTM you can Merge it.

File: styles/questions.css (hunk 1 above)
Comment (samyukthasudhakar) [reply styles/questions.css]: Changed it.

File: styles/questions.css (hunk 2 above)
Comment (samyukthasudhakar) [reply styles/questions.css:28]: Haven changed the former to match the latter's format to maintain consistency.

**Comment by samyukthasudhakar** [comment]
> Overall great work. not to sound patronizing, but your coding is evolving PR by PR. :)
> 
> One suggestion - leave consistence space between `property-name: value` . 💯

This means a lot, thank you for the feedback 😊
Changed code to maintain consistent spacing between prop and value.

**Pull request merged**